### PR TITLE
update nncf hooks

### PR DESCRIPTION
**Pull request by yunchu**
* added 'force=True' arg to hook registration decorator. it will allow registration of
  existing hook.

Signed-off-by: Yunchu Lee <yunchu.lee@intel.com>

### CODE REVIEW

File: mmdet/integration/nncf/compression_hooks.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 from mmcv.runner.dist_utils import master_only
 
 
-@HOOKS.register_module()
+@HOOKS.register_module(force=True)
```
Comment (AlexanderDokuchaev) [inline mmdet/integration/nncf/compression_hooks.py:9]: Please add comment with reason to add force argument, something like "force=true added to work with mmdet and mmseg in one environment"

Comment (yunchu) [reply mmdet/integration/nncf/compression_hooks.py:9]: do you mean that some comment should be on modified codes, right?

**Review by alexsu52 — COMMENTED** [review]
It looks good to me. Please check your changes on the BMM.

File: mmdet/integration/nncf/compression_hooks.py (hunk 1 above)
Comment (AlexanderDokuchaev) [reply mmdet/integration/nncf/compression_hooks.py:9]: Yes 
```suggestion
# Argument 'force' used to work with mmdetection and mmsegmentation repositories in one environment. 
@HOOKS.register_module(force=True)
```

**Comment by yunchu** [comment]
@alexsu52 can you give me some hint for the BMM?

**Review by AlexanderDokuchaev — APPROVED** [review]

**Pull request merged**